shaders/README.adoc

```diff
@@ -22,6 +22,10 @@
 
 This folder contains the textual shaders for the samples. All samples come with GLSL shaders and some optionally with HLSL shaders. For samples that support both shader language this is a good way to compare GLSL to HLSL syntax when targeting SPIR-V for Vulkan.
 
+== Compling shaders
```

Comment:
typo: Compiling

Reply:
Well, that's an easy fix :)

Thank you very much for your review 👍🏻